specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2023-01-01-preview/sqlvm.json

```diff
@@ -2168,7 +2168,7 @@
         },
         "enableStorageConfigBlade": {
           "description": "Enable SQL IaaS Agent storage configuration blade in Azure Portal.",
-          "default": "false",
+          "default": false,
```

Comment:
Now that it's type is boolean, it shall be `false` instead of `"false"` like https://github.com/Azure/azure-rest-api-specs/blob/14883286eb713f2ec2b969093c5f195b72c09b4a/specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2022-08-01-preview/sqlvm.json#L1884